src/test/java/org/gene/NumberOfBalloonsTest.java

```diff
@@ -0,0 +1,14 @@
+package org.gene;
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+
+@Slf4j
+public class NumberOfBalloonsTest {
+    @Test
+    void testMaxNumberOfBalloons() {
+        NumberOfBaloons solution = new NumberOfBaloons();
+        int result = solution.maxNumberOfBalloons("nlaebolko");
+        log.info("Result for testMaxNumberOfBalloons: {}", result);
```

Comment:
this test is always positive, No assertion. not prepared data no expected date. We cannot consider this as a good test case